.github/workflows/helm-lint-test.yaml

```
@@ -75,4 +75,15 @@ jobs:
 
       - name: Run chart integration tests
         if: steps.chart-list-changed.outputs.changed == 'true'
-        run: ct lint-and-install --config .github/configs/helm-lint-test.yaml
+        run: ct lint-and-install --config .github/configs/helm-lint-test.yaml --skip-clean-up
```

Suggestion: Please include this flag `skip-clean-up` directly inside the `.github/configs/helm-lint-test.yaml` instead of hard-coding here. 
```suggestion
        run: ct lint-and-install --config .github/configs/helm-lint-test.yaml
```